packages/mdc-tabs/tab/mdc-tab.scss

```diff
@@ -0,0 +1,198 @@
+/**
+* Copyright 2017 Google Inc. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+@import "@material/animation/functions";
+@import "@material/typography/mixins";
+@import "@material/theme/mixins";
+@import "@material/ripple/mixins";
+@import "@material/rtl/mixins";
+
+$mdc-tabs-text-color: rgba(black, .54);
+$mdc-tabs-active-text-color: rgba(black, .87);
+
+@mixin mdc-tabs-icon-tab-content {
+  display: block;
+  margin: 0 auto;
+}
+
+@mixin mdc-tabs-anchor-reset {
+  color: $mdc-tabs-text-color;
+  text-decoration: none;
+  cursor: pointer;
+
+  &:hover {
+    color: $mdc-tabs-active-text-color;
+  }
+
+  &:focus {
+    outline-color: $mdc-tabs-text-color;
+  }
+}
+
+// postcss-bem-linter: define tab
+.mdc-tab {
+  @include mdc-ripple-base;
+  @include mdc-ripple-bg((pseudo: "::before"));
+  @include mdc-ripple-fg((pseudo: "::after"));
+  @include mdc-tabs-anchor-reset;
+  @include mdc-typography(body2);
+
+  position: relative;
+  box-sizing: border-box;
+  display: table-cell;
```

Comment:
Awesome. It is possible not having `.mdc-tabs--icons-with-text .mdc-tab__icon` defined in `mdc-tab`? I think it should be handled directly by `flex` when you trying to align them center?